delfin/tests/unit/drivers/hitachi/vsp/test_hitachi_vspstor.py

```diff
@@ -0,0 +1,387 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from unittest import TestCase, mock
+
+from delfin import context
+from delfin.drivers.hitachi.vsp.rest_handler import RestHandler
+from delfin.drivers.hitachi.vsp.vsp_stor import HitachiVspDriver
+from delfin.drivers.utils.rest_client import RestClient
+from requests import Session
```

Comment:
Why move this import sentence to this line?

Reply:
changed